sklearn/impute.py

```diff
@@ -94,6 +103,13 @@ class SimpleImputer(BaseEstimator, TransformerMixin):
           each column.
         - If "most_frequent", then replace missing using the most frequent
           value along each column.
+        - If "constant", then replace missing values with fill_value.
+
+    fill_value : string or numerical value, optional (default=None)
```

Comment:
This is a bit confusing to state `default=None` since it will be either 0 or missing_value.
I would drop it since that the docstring below is more informative.